docs/release-notes/8.8.asciidoc

```diff
@@ -193,6 +233,47 @@ GET .kibana*/_search
 }
 ----------------------------------
 
+* A UI bug can affect the Alerts table's height and make it difficult to view alerts. View the alerts in Timeline or choose one of the following workarounds to readjust the table:
+
+** Use the Alerts table **Fields** browser to remove fields until the table width is smaller than its container.
+** Adjust your OS’ zoom settings and refresh the page.
+** Zoom your browser in and then out.
```

Comment:
Same as above

Reply:
Addressed in [14fa016](https://github.com/elastic/security-docs/pull/3644/commits/14fa016d290d8a68f16c88aa183458cfbfef3cae).